docs/source/rfcs/lts.rst

```diff
@@ -0,0 +1,358 @@
+
+ 55.0
+   * sprint release
+   * critical bug that affects 52.1 *only* is found, fix gets added to
+     52lts and 52.2 LTS is immediately released
+
+ 56.0
+  * sprint release
+
+ 57.0
+  * sprint release
+
+ 58.0
+  * sprint release
+
+ 59.0
+  * sprint release
+  * EOL for **36.x LTS** (18 months since the release of 36.0)
+
+Avocado-VT
+==========
+
+Avocado-VT is an Avocado plugin that allows "VT tests" to be run
+inside Avocado.  It's a third-party project maintained mostly by
+Engineers from Red Hat QE with assistance from the Avocado team
+and other community members.
+
+It's a general consensus that QE teams use Avocado-VT directly
+from git, usually following the master branch, which they
+control.
+
+There's no official maintenance or stability statement for
+Avocado-VT.  Even though the upstream community is quite
+friendly and open to both contributions and bug reports,
+Avocado-VT is made available without any promises for
+compatibility or supportability.
+
+When packaged and versioned, Avocado-VT rpms should be considered
+just snapshots, available in packaged form as a convenience to
+users outside of the Avocado-VT development community.  Again,
+they are made available without any promises of compatibility or
+stability.
+
+* Which Avocado version should be used by Avocado-VT?
+
+  This is up to the Avocado-VT community to decide, but the
+  current consensus is that to guarantee some stability in
+  production environments, Avocado-VT should stick to a specific
+  LTS release of Avocado. In other words, the Avocado team
+  recommends production users of Avocado-VT not to install Avocado
+  from its master branch or upgrade it from Sprint Releases.
+
+  Given each LTS release will be maintained for 18 months, it
+  should be reasonable to expect Avocado-VT to upgrade to a new
+  LTS release once a year or so. This process will be done with
+  support from the Avocado team to avoid disruptions, with proper
+  coordination via the avocado mailing lists.
+
+  In practice the Avocado development team will keep watching
+  Avocado-VT to detect and document incompatibilities, so when
+  the time comes to do an upgrade in production, it's expected
+  that it should happen smoothly.
+
+* Will it be possible to use the latest Avocado and Avocado-VT
+  together?
+
+  Users are welcome to *try* this combination.  The Avocado
+  development team itself will do it internally as a way to monitor
+  incompatibilities and regressions.
+
+  Whenever Avocado is released, a matching versioned snapshot of
+  Avocado-VT will be made.  Packages containing those Avocado-VT
+  snapshots, for convenience only, will be made available in the
+  regular Avocado repository.
+
+.. _Version 1: https://www.redhat.com/archives/avocado-devel/2016-April/msg00006.html
+.. _Version 2: https://www.redhat.com/archives/avocado-devel/2016-April/msg00038.html
+.. _Version 3: https://www.redhat.com/archives/avocado-devel/2017-April/msg00032.html
```

Comment:
I like this section, here is where you can get the development part with changes...